### DISCUSSION TITLE
Why the Space component is inline-flex by default?

### DISCUSSION THREAD

**Discussion by VladislavMaksimov** [ant-design/ant-design#44121] [Q&A] [answered]
Hello. I used MUI for one of the previous React projects and now I'm using Ant Design. I found it interesting that there is an essential difference in two analogous components in these UI Kits: [Space](https://ant.design/components/space) in Ant Design is `inline-flex` whereas [Stack](https://mui.com/material-ui/api/stack/) is just `flex`. So I'm interested to know the motivation to make a Space component `inline-flex`.

**Comment by zombieJ** [comment, marked as answer]
Space is design for inline align component which will also wrap with `ant-space-item`. For the block layout here is the RFC: https://github.com/ant-design/ant-design/discussions/44070
